src/core/TxInfo.spec.ts

```diff
@@ -59,21 +59,28 @@ const instantiateContractTxAmino = {
     body: {
       messages: [
         {
-          '@type': '/terra.oracle.v1beta1.MsgAggregateExchangeRateVote',
-          salt: '7b4c',
-          exchange_rates:
-            '41.753679997386404699uaud,38.152897754805427746ucad,27.674031355218179072uchf,196.026837309362459253ucny,191.737686262835618671udkk,25.782554425634391941ueur,22.070815310496596573ugbp,235.918907556073354996uhkd,2245.955815077995150686uinr,3326.162377566256916073ujpy,35308.691895711128492846ukrw,86357.377776669110190178umnt,267.642031750061676333unok,21.345339717339129734usdr,263.288481492217392139usek,41.027386540303457627usgd,997.082326305691696722uthb,30.291256499267926454uusd',
-          feeder: 'terra1fa0trn2nqjc2n6mmz9txta7ky5h5nnp9m6cra3',
-          validator: 'terravaloper1vk20anceu6h9s00d27pjlvslz3avetkvnwmr35',
+          '@type': '/cosmos.bank.v1beta1.MsgSend',
```

Comment:
since feather doesn't support classic, i think we can just use standard sdk modules to test here